app/src/main/resources/templates/thymeleaf/views/gene-search-results/gene-search-results_content.html

```diff
@@ -0,0 +1,35 @@
+<style>
+    #gene-search, #feedback-button {
+        display: inline;
+    }
+    #feedback-button h1 {
+        color: black;
+    }
+</style>
+
+<div class="row column expanded">
+    <div id="gene-search"></div>
+</div>
+
+<div id="feedback-button"></div>
+
+<link rel="stylesheet" th:href="@{/resources/css/react-popup.css}" type="text/css" media="all">
+<link rel="stylesheet" href="https://ebi.emblstatic.net/web_guidelines/EBI-Icon-fonts/v1.3/fonts.css" type="text/css" media="all" />
+
+<script defer th:src="@{/resources/js-bundles/feedbackForm.bundle.js}"></script>
+
+<script defer th:src="@{/resources/js-bundles/geneSearch.bundle.js}"></script>
+
+<script th:inline="javascript" th:with="contextPath=@{/}">
+  const contextPath = /*[[${contextPath}]]*/ '';
+  document.addEventListener("DOMContentLoaded", function() {
+    geneSearch.render({
+      atlasUrl: contextPath,
+      basename: contextPath,
+    }, 'gene-search');
+    feedbackForm.render({
+      feedbackFormLink: 'https://www.ebi.ac.uk/support/gxasc',
```

Comment:
should this change according to the env? (dev, test, prod)?

Reply:
You asked this already when I did `bulk`'s transformation ;-)
Unfortunately we don't have that URL by environments.
Precisely, there is none for `test` env and there is only a debug only version for `dev` with this message on the top of the page:
Debug mode! - Messages submitted via this form will NOT be processed, please use [www.ebi.ac.uk/about/contact/support/gxasc](https://www.ebi.ac.uk/about/contact/support/gxasc) if you have landed on this page by mistake.